GVFS/GVFS/CommandLine/MountVerb.cs

```diff
@@ -294,12 +290,6 @@ private bool TryMount(GVFSEnlistment enlistment, string mountExecutableLocation,
                     this.StartedByService.ToString()
                 });
 
-            if (GVFSPlatform.Instance.IsUnderConstruction)

```

Comment:
Not really sure what we've done that fixed this. I remember adding this sleep, and back then connecting to the pipe without sleeping first would fail most of the time. Now it is succeeding reliably. My only guess is that since we've changed the location of the socket file, it has also changed the timing somewhat.

Reply:
It might have also been fixed when we moved to a more recent version of .NET Core